tests/test_source_sampling.py

```diff
@@ -41,10 +41,10 @@ def try_rm_file(filename):
 
 
 @with_setup(None, try_rm_file('tet.h5m'))
-def test_single_tet_tag_names_map():
```

Comment:
Is this for hex or for tet?

Reply:
The old name of this function indicates that it is for `tet`. However, the mesh used inside the function is `hex`. Therefore, I changed the name of this function from `tet` to `hex`.

Reply:
Maybe we should change the file name as well from `tet.h5m` to `hex.h5m`